examples/kubernetes/env.sh

```diff
@@ -15,7 +15,8 @@ KUBERNETES_API_SERVER=${KUBERNETES_API_SERVER:-'127.0.0.1:8080'}
 VITESS_NAME=${VITESS_NAME:-'default'}
 
 # Kubernetes options config
-KUBECTL_OPTIONS="--namespace=$VITESS_NAME --server=$KUBERNETES_API_SERVER"
+#KUBECTL_OPTIONS="--namespace=$VITESS_NAME --server=$KUBERNETES_API_SERVER"
+KUBECTL_OPTIONS="--namespace=$VITESS_NAME"
```

Comment:
I think Josh fixed that in https://github.com/youtube/vitess/pull/2727/files so you might want to revert this file.